### PR TITLE
Trending environments

### PR DESCRIPTION
**Pull request by maxidragon**

### CODE REVIEW

File: src/renderer/src/Layout/Header.tsx
Comment (Majkipl27) [inline src/renderer/src/Layout/Header.tsx:1]: Please remove also the navlink in the responsive view.

**Review by Majkipl27 — CHANGES_REQUESTED** [review]

File: src/renderer/src/Pages/Profile/Components/Environment.tsx [hunk 1]
```diff
@@ -196,26 +202,108 @@ export default function Environment({
...
+            {loggedUserData.id === environment.user.id &&
+              setEditedEnvironmentId &&
+              setIsUserEditingEnvironment && (
+                <>
+                  <TbEdit
+                    onClick={() => {
+                      setEditedEnvironmentId(environment.id);
+                      setIsUserEditingEnvironment(true);
+                    }}
+                    title="Edit ruleset"
```
Comment (maxidragon) [inline src/renderer/src/Pages/Profile/Components/Environment.tsx:234]:
```suggestion
                    title="Edit environment"
```

File: src/renderer/src/Pages/Profile/Components/Environment.tsx [hunk 2]
```diff
@@ -196,26 +202,108 @@ export default function Environment({
...
+                    className="cursor-pointer text-borderGray hover:text-gray-500 transition-all"
+                  />
+                  <TbTrash
+                    onClick={deleteEnv}
+                    title="Delete ruleset"
```
Comment (maxidragon) [inline src/renderer/src/Pages/Profile/Components/Environment.tsx:239]:
```suggestion
                    title="Delete environment"
```

File: src/renderer/src/Pages/SavedEnvironments/SavedEnvironments.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import { motion } from "framer-motion";
+import { useEffect, useState } from "react";
+import toast from "react-hot-toast";
+import Environment from "../Profile/Components/Environment";
+import { TbLayoutGrid, TbLayoutList } from "react-icons/tb";
+import { userAtom } from "@renderer/lib/atoms";
+import { useAtomValue } from "jotai";
+import { useNavigate } from "react-router-dom";
+
+export default function SavedEnvieonments(): JSX.Element {
```
Comment (maxidragon) [inline src/renderer/src/Pages/SavedEnvironments/SavedEnvironments.tsx]:
```suggestion
export default function SavedEnvironments(): JSX.Element {
```

Remember to change all references too

File: src/renderer/src/Pages/Profile/Components/Environment.tsx [hunk 4]
```diff
@@ -165,7 +182,9 @@ export default function Environment({
               </p>
             )}
             <span
-              className="flex items-center justify-end gap-2 cursor-pointer"
+              className={`${
```
Comment (maxidragon) [inline src/renderer/src/Pages/Profile/Components/Environment.tsx]: Please use `twMerge` function

File: src/renderer/src/Pages/Profile/Components/Environment.tsx [hunk 5]
```diff
@@ -248,16 +284,18 @@ export default function Environment({
...
-            className="hover:underline text-xl font-roboto text-gray-400"
+            className="hover:underline text-xl font-roboto text-gray-400 mr-auto"
             title="Environment details"
             to={`/environments/${environment.id}`}
           >
             {environment.name}
           </Link>
           <p
-            className="flex items-center justify-end gap-2 cursor-pointer"
+            className={`${
```
Comment (maxidragon) [inline src/renderer/src/Pages/Profile/Components/Environment.tsx]: See above

**Comment by maxidragon** [comment]
LGTM

**Pull request merged**